Doc/library/typing.rst

```diff
@@ -1230,6 +1232,123 @@ These are not used in annotations. They are building blocks for creating generic
     ``covariant=True`` or ``contravariant=True``.  See :pep:`484` for more
     details.  By default, type variables are invariant.
 
+.. class:: TypeVarTuple
+
+    Type variable tuple. A specialised form of :class:`Type variable <TypeVar>`
+    that enables *variadic* generics.
+
+    A normal type variable enables parameterization with a single type. A type
+    variable tuple, in contrast, allows parameterization with an
+    *arbitrary* number of types by acting like an *arbitrary* number of type
+    variables wrapped in a tuple. For example::
+
+        T = TypeVar('T')
+        Ts = TypeVarTuple('Ts')
+
+        def remove_first_element(xs: tuple[T, *Ts]) -> tuple[*Ts]:
+            return xs[1:]
+
+        # T is bound to int, Ts is bound to ()
+        # Return value is (), which has type tuple[()]
+        remove_first_element(xs=(1,))
+
+        # T is bound to int, Ts is bound to (str,)
+        # Return value is ('foo',), which has type tuple[str]
```

Comment:
This typo's on me

```suggestion
        # Return value is ('spam',), which has type tuple[str]
```

Reply:
lol, sorry, shoulda checked more carefully